torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1421,6 +1426,22 @@ def sample_inputs_masked_select(op_info, device, dtype, requires_grad):
            supports_tensor_out=False,
            test_inplace_grad=False,
            sample_inputs_func=sample_inputs_broadcast_to),
+    UnaryUfuncInfo('bitwise_not',
+                   ref=np.bitwise_not,
+                   dtypes=integral_types_and(torch.bool),
```

Comment:
It's humorous that this doesn't need to specify that it's not differentiable since it only supports integer types.